storage/database/src/main/java/de/danoeh/antennapod/storage/database/DBUpgrader.java

```diff
@@ -330,6 +330,10 @@ static void upgrade(final SQLiteDatabase db, final int oldVersion, final int new
             db.execSQL("ALTER TABLE " + PodDBAdapter.TABLE_NAME_FEED_ITEMS
                     + " ADD COLUMN " + PodDBAdapter.KEY_PODCASTINDEX_CHAPTER_URL + " TEXT");
         }
+        if (oldVersion < 3010000) {
```

Comment:
@ByteHamster I chose this DB version arbitrarily, I'm not sure if there is a convention to how the version is integrated. Let me know if this should be changed before merge.